lib/nif.ex

```diff
@@ -5,11 +5,42 @@ defmodule Mediasoup.Nif do
 
   version = Mix.Project.config()[:version]
 
+  defmodule Build do
+    def prebuild_targets() do
+      # Anything that is difficult to prebuilt in cross compile is excluded for now.
+      [
+        #  "aarch64-apple-darwin",
+        "aarch64-unknown-linux-gnu",
+        "aarch64-unknown-linux-musl",
+        "arm-unknown-linux-gnueabihf",
+        "riscv64gc-unknown-linux-gnu",
+        "x86_64-apple-darwin",
+        #    "x86_64-pc-windows-gnu",
+        #    "x86_64-pc-windows-msvc",
+        "x86_64-unknown-linux-gnu",
```

Comment:
In my local, I tried to export `RustlerPrecompiled.target()` and that is `{:ok, "nif-2.16-x86_64-unknown-linux-gnu"}`. So, the precompiled build is not used.